src/core_plugins/tile_map/public/maps_visualization.js

```diff
@@ -261,10 +272,47 @@ export function MapsVisualizationProvider(serviceSettings, Notifier, getAppState
       this.vis.updateState();
     }
 
-  }
+    async getGeohashBounds() {
```

Comment:
The function is now passed to the geohash_layer. I did this instead of migrating the code to geohash_layer because several dependencies are not available in geohash_layer, like Private and vis index pattern.